packages/shared/src/components/post/write/SquadsDropdown.tsx

```diff
@@ -2,30 +2,42 @@ import React, { ReactElement } from 'react';
 import { SourceAvatar, SourceShortInfo } from '../../profile/source';
 import { ArrowIcon, SquadIcon } from '../../icons';
 import { Dropdown } from '../../fields/Dropdown';
-import { useAuthContext } from '../../../contexts/AuthContext';
-import { verifyPermission } from '../../../graphql/squads';
-import { SourcePermissions } from '../../../graphql/sources';
+import { SourceMemberRole, SourceType, Squad } from '../../../graphql/sources';
 import { ButtonSize } from '../../buttons/common';
 import { useViewSize, ViewSize } from '../../../hooks';
+import { cloudinary } from '../../../lib/image';
 
 interface SquadsDropdownProps {
   onSelect: (index: number) => void;
   selected: number;
+  list: Squad[];
 }
 
+export const generateDefaultSquad = (username: string): Squad => ({
+  id: username,
+  handle: username,
```

Comment:
More wondering what happens frontend wise when you accidentally create a duplicate id/handle squad?
You get the handle toast message, but then what?

Reply:
If I remember correctly, from the RFC, it was decided to not cover the edge case as it is a bit unique. However, I will verify and get back to this.